### PR TITLE
CfP: an attempt at clearer, friendlier intro

### PR DESCRIPTION
**Pull request by shlomi-noach**
This PR modifies the intro text a bit, in an attempt to make it:

- More inviting intro text (first paragraph in intro)
- Clearer topics: we have 2*categories, 1st in intro text, 2nd in proposal itself, and they are incompatible. The 1st makes the appearance we accept a smaller range of topics than we truly do.
- Reversed summit history list; 2017 is more relevant than 2013.

I'm happy to hear your thoughts/comments.

### CODE REVIEW

File: client/src/components/CFPIntro.js [hunk 1]
```diff
@@ -6,37 +6,36 @@ const CFPIntro = () => (
   <div className="pb-6 mb-8 border-bottom">
     <h2 className="mb-4">Intro</h2>
     <h4>Welcome to Reversim Summit and thank you for submitting your proposal!</h4>
-    <p className="mb-8">Our goal is to streamline the submission and review process, while maintaining superb quality; if you have any feedback or questions, please email us at <a href={`mailto:${REVERSIM_MAIL}`}>{REVERSIM_MAIL}</a>.</p>
+    <p>
+      We encourage all people to submit a proposal to Reversim Summit 2018. If you are _considering_ submitting a proposal, we invite you to do so!
```
Comment (rantav) [inline client/src/components/CFPIntro.js]: Phrasing: suggest to replace `all people` with `everyone`

Comment (rantav) [inline client/src/components/CFPIntro.js]: geekness question? Replace `_considering_` with `<i>considering</i>`? Not sure, it's nice to geek at time...

Comment (shlomi-noach) [reply client/src/components/CFPIntro.js]: LOL that wasn't intentional.

Comment (shlomi-noach) [reply client/src/components/CFPIntro.js]: done

Comment (shlomi-noach) [reply client/src/components/CFPIntro.js]: done

**Comment by shlomi-noach** [comment]
Request to review changes please. No urgency.

**Comment by shlomi-noach** [comment]
Friendly ping.

**Comment by shlomi-noach** [comment]
I'm pinging @rantav who has requested changes on your review, changes which I have made. I should probably just dismiss your review, but I'm too polite to do that.

**Pull request merged**